File: Src/JS/Connect to Wi-Fi.js
ESP8266_IoT.initWIFI(SerialPin.P8, SerialPin.P12, BaudRate.BaudRate115200)
ESP8266_IoT.connectWifi("YOUR WIFI", "AND ITS PASSWORD")
basic.forever(function on_forever() {
    if (ESP8266_IoT.wifiState(true)) {
        basic.showLeds(`
            # # # # #
            # # # # #
            # # # # #
            # # # # #
            # # # # #
            `)
        basic.clearScreen()
        basic.pause(1000)
    }
    
})

File: Src/Python/Connect to Wi-Fi.py
ESP8266_IoT.init_wifi(SerialPin.P8, SerialPin.P12, BaudRate.BAUD_RATE115200)
ESP8266_IoT.connect_wifi("YOUR WIFI", "AND ITS PASSWORD")

def on_forever():
    if ESP8266_IoT.wifi_state(True):
        basic.show_leds("""
            # # # # #
            # # # # #
            # # # # #
            # # # # #
            # # # # #
            """)
        basic.clear_screen()
        basic.pause(1000)
basic.forever(on_forever)
